android/src/com/frostwire/android/gui/fragments/preference/OtherFragment.java

```diff
@@ -110,7 +110,7 @@ private void setupClearIndex() {
                 @Override
                 public void onClick(View v) {
                     LocalSearchEngine.instance().clearCache();
-                    UIUtils.showShortMessage(getActivity(), R.string.deleted_crawl_cache);
```

Comment:
ok, so it's a compatibility issue? meaning, those toasts need something loaded before hand?
I'd think toasts would be sort of independent windows.
I'll have to test the branch myself tomorrow.
If I make it work, I'll submit a PR to greg on this branch.